examples/01-blog/graphql.php

```diff
@@ -1,74 +1,71 @@
 <?php
-// Test this using following command
-// php -S localhost:8080 ./graphql.php
+
+declare(strict_types=1);
+
+// Test this using the following command:
+// php -S localhost:8080 graphql.php
+
 require_once __DIR__ . '/../../vendor/autoload.php';
 
+use GraphQL\Error\DebugFlag;
+use GraphQL\Error\FormattedError;
+use GraphQL\Examples\Blog\AppContext;
+use GraphQL\Examples\Blog\Data\DataSource;
 use GraphQL\Examples\Blog\Type\QueryType;
-use \GraphQL\Examples\Blog\Types;
-use \GraphQL\Examples\Blog\AppContext;
-use \GraphQL\Examples\Blog\Data\DataSource;
-use \GraphQL\Type\Schema;
-use \GraphQL\GraphQL;
-use \GraphQL\Error\FormattedError;
-use \GraphQL\Error\DebugFlag;
+use GraphQL\Examples\Blog\Types;
+use GraphQL\GraphQL;
+use GraphQL\Type\Definition\Type;
+use GraphQL\Type\Schema;
 
-// Disable default PHP error reporting - we have better one for debug mode (see below)
-ini_set('display_errors', 0);
+// Replace default PHP error reporting with a better handler
+ini_set('display_errors', '0');
+set_error_handler(static function ($severity, $message, $file, $line): void {
```

Comment:
Maybe this can be dropped entirely? PHP error handling is out of scope. And if we don't want to drop it, it's not clear why it's a "better handler" 🧐

Reply:
I thought about just using the standard server for this example. We have `00-hello-world` as an example of handling the HTTP aspect ourselves, but I would not recommend anyone to actually do this in a real application.

Reply:
So if we want to keep this, maybe we should specify why it is better handler. 🤷🏾

Reply:
I don't actually know.

Reply:
hm, me neither :D umm

Reply:
I switched to StandardServer.